pkgs/jni/lib/src/accessors.dart

```diff
@@ -59,7 +59,8 @@ extension JniResultMethods on JniResult {
   }
 
   JReference get reference {
-    return JGlobalReference(objectPointer);
+    final pointer = objectPointer;
+    return pointer == nullptr ? jNullReference : JGlobalReference(pointer);
```

Comment:
Not related to this PR, but have you considered using Dart's `null` instead of `jNullReference`? That's what the ObjC bindings do. It means you get Dart's null safety for free.

Reply:
Every Java object *can* be null, so it creates a lot of question mark noise. Could be an option though.

Reply:
Oh wait you mean only for the reference type? Yeah I could do that!